apps/src/blockly/customBlocks/cdoBlockly/spritelabBlocks.js

```diff
@@ -0,0 +1,55 @@
+export const blocks = {
```

Comment:
Can you add a top level comment explaining the purpose of this file? Similarly for the other new files.

Reply:
Sure! Will do.